test/cases/oscap.sh

```diff
@@ -480,7 +486,7 @@ fi
 
 # one grub rule fails (expected)
 # check if any other rules have failed
-if [[ ${SEVERITY} -gt 1 ]]; then
+if [[ ${SEVERITY} -gt 0 ]]; then
   redprint "❌ Failed"
   echo "More than one oscap rule with high severity failed"
```

Comment:
Wording needs to be updated as well.

Reply:
I guess this is still an issue?

Reply:
Ah flip, I must have reverted the changes when I lost the commits

Reply:
Ah no the diff shows it's been removed. But it's not showing on the review comment weirdly enough.

Reply:
Ah flip haha I'm blind. I was too focussed on the diff